storages/backends/ftp.py

```diff
@@ -51,13 +53,14 @@ def __init__(self, location=None, base_url=None, encoding=None):
         self._config = self._decode_location(location)
         self._base_url = base_url
         self._connection = None
+        self._secure = secure
 
     def _decode_location(self, location):
         """Return splitted configuration data from location."""
         splitted_url = urlparse(location)
         config = {}
 
-        if splitted_url.scheme not in ("ftp", "aftp"):
+        if splitted_url.scheme not in ("ftp", "aftp", "ftps"):
```

Comment:
Please set `config[“secure”]` based on if the url is `ftps`.

Reply:
Thanks for the suggestions @jschneier.

I'll modify them add suggested and minimize the tests as instructed.